src/class-wpml-gutenberg-integration.php

```diff
@@ -258,4 +261,27 @@ public function wpml_config_filter( $config_data ) {
 		return $config_data;
 	}
 
+	/**
+	 * @param bool    $bool
+	 * @param WP_Post $post
+	 * @param string  $context
+	 *
+	 * @return bool
+	 */
+	public function should_body_be_translated_filter( $bool, WP_Post $post, $context = '' ) {
```

Comment:
@strategio I would rename `$bool` to `$translate`

Reply:
changed @brucepearson (I was not inspired 😀)